_opensearch/query-dsl/geo-bounding-box.md

```diff
@@ -0,0 +1,226 @@
+
+## Specifying the bounding box
+
+You can specify the bounding box by providing any of the following combinations of its vertex coordinates:
+
+- `top_left` and `bottom_right`
+- `top_right` and `bottom_left`
+- `top`, `left`, `bottom`, and `right`
+
+The following example specifies the bounding box using the `top`, `left`, `bottom`, and `right` coordinates:
+
+```json
+GET testindex1/_search
+{
+  "query": {
+    "bool": {
+      "must": {
+        "match_all": {}
+      },
+      "filter": {
+        "geo_bounding_box": {
+          "point": {
+            "top": 75,
+            "left": 28,
+            "bottom": 73,
+            "right": 41            
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+## Request fields
+
+Geo-bounding box queries accept the following fields.
+
+Field | Data type | Description
+:--- | :--- | :--- 
+_name | String | The name of the filter. Optional.
+validation_method | String | The validation method. Valid values are `IGNORE_MALFORMED` (accept geopoints with invalid coordinates), `COERCE` (try to coerce  coordinates to valid values), and `STRICT` (return an error when coordinates are invalid). Default is `STRICT`.
+type | String | Specifies how to execute the filter. Valid values are `indexed` (index  the filter) and `memory` (execute the filter in memory). Default is `memory`.
+ignore_unmapped | Boolean | Specifies whether to ignore an unmapped field. If set to `true`, the query does not return any documents that have an unmapped field. If set to `false`, an exception is thrown when the field is unmapped. Default is `false`.
+
+## Accepted formats
+
+You can specify coordinates of the bounding box vertices in any [format]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-point#formats) that geopoint accepts.  
+
+### Using geohash to specify the bounding box
+
+If you use a geohash to specify the bounding box, the geohash is treated as a rectangle. The upper left vertex of the bounding box corresponds to the upper left vertex of the `top_left` geohash, and the lower right vertex of the bounding box corresponds to the lower right vertex of the `bottom_right` geohash. 
+
+The following example uses a geohash to specify the same bounding box as the previous examples:
+
+```json
+GET testindex1/_search
+{
+  "query": {
+    "bool": {
+      "must": {
+        "match_all": {}
+      },
+      "filter": {
+        "geo_bounding_box": {
+          "point": {
+            "top_left": "ut7ftjkfxm34",
+            "bottom_right": "uuvpkcprc4rc"
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+To specify the bounding box that covers the whole area of a geohash, provide that geohash as both `top_left` and `bottom_right` parameters of the bounding box:
```

Comment:
"To specify a bounding box"? "the" is correct if there is just one preexisting bounding box that covers the whole area.